core/src/main/java/io/appflate/restmock/utils/RequestMatchers.java

```diff
@@ -47,19 +47,24 @@ protected boolean matchesSafely(RecordedRequest item) {
         };
     }
 
-    public static RequestMatcher pathEndsWith(final String urlPart) {
-        return new RequestMatcher("url ends with: " + urlPart) {
-
-            @Override
+    public static RequestMatcher pathEndsWith(final String endOfUrlPath) {
```

Comment:
this changes logic underneath and breaks backward compatibility, maybe `pathEndsWithIgnoringQueryParams`?